recipes/approvaltests.cpp/8.0.0/conanfile.py

```diff
@@ -0,0 +1,42 @@
+import os
+from conans import ConanFile, tools
+
+
+class ApprovalTestsCppConan(ConanFile):
+    name = "approvaltests.cpp"
+    version = "8.0.0"
```

Comment:
Do not add the version here, it will be passed from the command line when generating the packages (that way, we can reuse this recipe for other versions)

```suggestion
```

Reply:
I see no problem when is a very specific folder version. Otherwise, we can use the folder name `all` and remove the version from the recipe.

Reply:
You are right, @uilianries . I didn't realize it was in a specific folder. Probably the same recipe is valid for other versions and we could move it to a `all` folder. 👍